File: iiwa floating/xmat0.m
function X = xmat0(theta)
    X = [cos(theta), sin(theta), 0, 0, 0, 0;
         -sin(theta), cos(theta), 0, 0, 0, 0;
         0, 0, 1, 0, 0, 0;
         -0.1575 * sin(theta), 0.1575 * cos(theta), 0, cos(theta), sin(theta), 0;
         -0.1575 * cos(theta), -0.1575 * sin(theta), 0, -sin(theta), cos(theta), 0;
         0, 0, 0, 0, 0, 1];
end

File: RNEA_GPU_Parallelization-main/xmat_files/xmat0.py
from numpy import array, sin, cos

def _lambdifygenerated(theta):
    return array([[1.0*cos(theta), 1.0*sin(theta), 0, 0, 0, 0], [-1.0*sin(theta), 1.0*cos(theta), 0, 0, 0, 0], [0, 0, 1.0, 0, 0, 0], [-0.1575*sin(theta), 0.1575*cos(theta), 0, 1.0*cos(theta), 1.0*sin(theta), 0], [-0.1575*cos(theta), -0.1575*sin(theta), 0, -1.0*sin(theta), 1.0*cos(theta), 0], [0, 0, 0, 0, 0, 1.0]])

